packages/suite/package.json

```diff
@@ -65,7 +65,7 @@
         "@trezor/suite-storage": "workspace:*",
         "@trezor/type-utils": "workspace:*",
         "@trezor/urls": "workspace:*",
-        "@trezor/utils": "workspace:*",
+        "@trezor/utils": "workspace:^",
```

Comment:
Why this change? It does not seem related.

Reply:
Not sure, when I added the package Typescript was complaining that it is not in the package so I installed it with yarn to suite but seems to not be needed...

Reply:
Maybe you just need to call `yarn build:libs` and it will work. You are not importing anything from the package, so I don't think the change is necessary. Also make sure `yarn.lock` does not have any changes.

Reply:
Yes, thanks, putted back